lib/northbound_db.c

```diff
@@ -169,9 +174,10 @@ struct nb_config *nb_db_transaction_load(uint32_t transaction_id)
 	if (db_loadf(ss, "%s", &config_str) != 0)
 		goto exit;
 
-	dnode = lyd_parse_mem(ly_native_ctx, config_str, LYD_XML,
-			      LYD_OPT_CONFIG);
-	if (!dnode)
+	err = lyd_parse_data_mem(ly_native_ctx, config_str, LYD_XML,
```

Comment:
nit: update function name in the `flog_warn()` call below

Reply:
fixed